src/haddock/modules/analysis/contactmap/defaults.yaml

```diff
@@ -0,0 +1,96 @@
+ca_ca_dist_threshold:
+  default: 9.0
+  min: 4.0
+  max: 1000.0
+  type: float
+  precision: 1
+  title: Ca-Ca distance defining a contact.
+  short: Distance between Ca atoms of two residues defining a contact.
+  long: Distance between Ca atoms of two residues defining a contact.
+  group: analysis
+  explevel: easy
+
+shortest_dist_threshold:
+  default: 7.5
+  type: float
+  min: 2.0
+  max: 1000.0
+  precision: 1
+  title: Distance defining a contact between two residues.
+  short: Shortest distance between any atom of two residues defining a contact.
+  long: Shortest distance between any atom of two residues defining a contact.
+  group: analysis
+  explevel: easy
+
+color_ramp:
+  default: 'Greys'
+  type: string
+  minchars: 3
+  maxchars: 10
+  choices:
+    - Greys
+    - Purples
+    - BuPu
+    - Blues
+    - BuGn
+    - Greens
+    - deep
+    - tempo
+    - algae
+    - dense
+    - amp
+  title: Color ramp used to color heatmap.
+  short: Color ramp used to color heatmap.
+  long: Please refere to `plotly builtin-colorscales` to get hints of the style of accessible colors.
+  group: analysis
+  explevel: easy
+
+single_model_analysis:
+  default: false
+  type: boolean
+  title: Analyse contacts at single model level.
+  short: Analyse contacts at single model level.
+  long: In addition to cluster analysis, also perform single model analyses.
+  group: analysis
+  explevel: easy
+
+generate_heatmap:
+  default: true
+  type: boolean
+  title: Generate contact heatmap.
+  short: Generate plotly graphical representation of the contacts.
+  long: Uses generated contact matrices files to generate heatmap.
+  group: analysis
+  explevel: easy
+
+topX:
+  default: 10
+  type: integer
+  min: 1
+  max: 100
+  title: Number of unclustered models to analyse.
+  short: Number of unclustered models to analyse.
```

Comment:
In means in case of unclustered models (`model.clu_id == None`), how many of them should be analysed, prioritizing the ones with best scores.
Otherwise all clusters are analysed.
Can be an issue